ci/docker/common.yaml

```diff
@@ -39,6 +39,9 @@ packages:
   hwloc:
     variants:
       - '~libxml2'
+  dla-future:
+    require:
+      - '@master'
```

Comment:
For testing before PR is merged or permanent? We should aim to get DLA-Future 0.7.0 as soon as possible so that you don't have to use master here.

Reply:
This is mainly for testing. I don't mind either merging now, or waiting for DLA-Future `0.7.0` release. In any case, a new release of DLA-Future-Fortran is conditional on the DLA-Future release, and therefore `@master` will become `@0.7.0` in the release.

Reply:
Thanks, that sounds good! Up to you to decide which way to do it, but probably easier to merge with the `@master` requirement for now and then update once 0.7.0 is out (to avoid dragging out this PR)?

Reply:
I don't mind either way. @rasolca seemed keen to get a DLAF release out soon, but we can definitely merge this now so that I can finalize #8 and then get rid of `@master` as soon as the new DLAF release is out.

Reply:
0.7.0 and 0.7.1 are out now.

Reply:
This is updated as part of #27.